changelogs/fragments/team-org-support.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - Enable option `org_id` for grafana_folder
+  - Enable option `org_id` for grafana_team
```

The changelog uses reStructuredText BTW, not MarkDown, so you need double backticks:
```suggestion
  - Enable option ``org_id`` for grafana_folder
  - Enable option ``org_id`` for grafana_team
```